test/NHSD.BuyingCatalogue.Documents.API.Integration.Tests/Steps/AzureBlobSteps.cs

```diff
@@ -6,9 +6,10 @@
 namespace NHSD.BuyingCatalogue.Documents.API.IntegrationTests.Steps
 {
     [Binding]
-    class AzureBlobSteps
+    internal class AzureBlobSteps
     {
         private AzureBlobStorageScenarioContext context;
```

Comment:
This field can be made `readonly` and while I don't like underscores in field names it probably ought to have one in this case to remain consistent with the rest of the project.

Reply:
👍